patchwork/patchflows/ResolveIssue/ResolveIssue.py

```diff
@@ -0,0 +1,51 @@
+from pathlib import Path
+
+import yaml
+
+from patchwork.step import Step
+from patchwork.steps import (
+    CreateIssueComment,
+    GenerateCodeRepositoryEmbeddings,
+    QueryEmbeddings,
+    ReadIssues,
+)
+
+_DEFAULT_INPUT_FILE = Path(__file__).parent / "defaults.yml"
+
+
+class ResolveIssue(Step):
+    def __init__(self, inputs: dict):
+        final_inputs = yaml.safe_load(_DEFAULT_INPUT_FILE.read_text())
+        if final_inputs is None:
+            final_inputs = {}
+        final_inputs.update(inputs)
+
+        self.inputs = final_inputs
+
+    def run(self) -> dict:
+        outputs = GenerateCodeRepositoryEmbeddings(self.inputs).run()
+        self.inputs.update(outputs)
+
+        outputs = ReadIssues(self.inputs).run()
+        self.inputs.update(outputs)
+
+        self.inputs["texts"] = self.inputs["issue_text"]
+
+        outputs = QueryEmbeddings(self.inputs).run()
+        self.inputs.update(outputs)
+
+        issue_list_test = "* " + "\n* ".join([result["path"] for result in self.inputs["embedding_results"]])
```

Comment:
Why do we call it `issue_list_test` can it be `issue_list`?

Reply:
or did you mean `issue_text_list`?

Reply:
its actually supposed to be `issue_list_text`.